### ISSUE TITLE
Starblade Alpha large STRs have no video

### ISSUE THREAD

**Issue by aybe** [m35/jpsxdec#24]
Environment
-------------

jPSXdec version: jpsxdec_v1-00_rev3921
Operating system: Microsoft Windows [Version 10.0.18362.418]
Java runtime environment java version "1.8.0_231"

 PlayStation Game (if applicable)
----------------------------------

Name: Starblade Alpha
Region code (e.g. SLUS-1234): SLUS-00057

 Description 
-------------

What were you doing?:

Convert FLT\GAME1.STR

What happened?:

There's only audio, no video

What was or was not supposed to happen?:

There should be video too

[index.log](https://github.com/m35/jpsxdec/files/3849568/index.log)
[debug00.log](https://github.com/m35/jpsxdec/files/3849569/debug00.log)

Thanks!

**Comment by aybe** [comment]
Also, it happens too on its suite, Galaxian 3 - SLPS-00270.

**Comment by m35** [comment]
Startblade Alpha support added https://github.com/m35/jpsxdec/releases/tag/v1.01

**Comment by aybe** [comment]
Amazing, thank you!!!

**Comment by m35** [comment]
By the way, it appears both Starblade Alpha and Galaxian 3 use the same special video format. I found it very curious that you happened to report both in this issue. Do you know something I don't? 😉 
And are there other games you think might use the same format?

**Comment by aybe** [comment]
Not really, unfortunately. It just happens that I have both games and naturally decided to try both!

A rough guess would be that:
- it's the second installment for PS1
- it's been released only 12 months later
- games are similar, recorded off Laserdisc

They certainly re-used most of the code.

Now as on why ...

I tried to diff out between old and last release to try understand what you've added, indeed there are new things but since I don't really know the subject it didn't really help. By the way, why didn't you commit directly to GitHub, it would be great to have an history of commits/versions/tags?

Other games? I don't know, possibly!

Instead, I looked at the docs @ http://www.psxdev.net/downloads.html:
- Technical Reference CD 2.3
  - :\Progcd\PSXGRAPH\BIN\mc32.pdf
  - :\TRAINING\Summer97\mfmovi.pdf (movie as a texture ?)
- Programmers Tool CD 2.2
  - :\PSXGRAPH\BIN\MC32.EXE

There seems to be a way to do custom interleaving, just a guess, Namco being a 1st class customer for Sony, they might had access to some undocumented features too.

I learned many things through these 2 ISOs from SDK, such as (mostly) properly reading Ridge Racer and Wipeout graphics:

This first game ever for the console, with PSX.EXE as executable like many early ones, didn't find anything for months, then suddenly I realized its structures mapped to most of what's in the docs.

![Unity_2019-04-16_22-33-24](https://user-images.githubusercontent.com/1537591/69031761-274bfa80-09db-11ea-8154-2a42cff0966a.png)

This one is more Western-style, easy to read game files, still, docs were helpful in understanding and getting proper rendering.

![Unity_2019-08-09_02-27-11](https://user-images.githubusercontent.com/1537591/69031775-2d41db80-09db-11ea-8bec-bc92c015aa5a.png)

Also, you certainly know https://problemkaputt.de/psx.htm, its GPU debugger has been really helpful to me.

Hope this helps, let's see if these docs can make you draw some conclusions and hopefully some mysteries will be uncovered :)
